### ISSUE TITLE
Exiting Tab Navigation

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#10382]
### Description

We should try to answer the question of: _Is it possible to exit tab navigation and reset it by pressing Esc or when clicking on something?_

If we do have such possibility, we need to understand where the focus should be redirected. In the navigation system, there is always a control that is in focus. But it is important that this control may not be highlighted, but will remain in focus, and when the User presses left or right, the next element will be highlighted.

**Comment by chinakov** [comment]
As per @Tantacrul - if 'Esc' key is pressed while a UI element is focused, we should bring the focus back to 'Home', which is the first focused element when 'Tab' key is pressed on launch.
